Use Path.expand to specify script in config.

suazithustra opened the pull request:
Some environments will read the lone string as bin/ch command without explicitly expanding on the path.

somlor commented:
@chvanikoff i didn't actually mean to submit this PR against your fork, but i'll leave it and out of curiousity for your feedback. feel free to close/ignore.

hafizio commented:
@somlor I'm currently facing this exact problem. Is there any fork where this solution is being implemented? Thanks

somlor commented:
@hafizio unfortunately we weren't able to solve the issues with this SSR approach in a timely enough manner and ended up backing away entirely. 😞

suazithustra committed:
Use Path.expand to specify script in config.

Some environments will read the lone string as bin/ch command without explicitly expanding the path.
